### PR TITLE
Hide question links for linked apps

### PR DESCRIPTION
**Pull request by proteusvacuum**
https://dimagi-dev.atlassian.net/browse/HI-501

FF affected: linked apps

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/form_summary_base.html [hunk 1]
```diff
@@ -65,7 +65,10 @@ <h5>
                 <ol data-bind="foreach: questions">
                     <li data-bind="visible: isVisible">
                         <i data-bind="attr: { 'class': $root.questionIcon($data), 'title': type }"></i>
+                        <!-- ko ifnot: $root.readOnly -->
```
Comment (emord) [inline corehq/apps/app_manager/templates/app_manager/form_summary_base.html]: root.readOnly is only true for linked apps?

File: corehq/apps/app_manager/templates/app_manager/form_summary_base.html [hunk 2]
```diff
@@ -65,7 +65,10 @@ <h5>
...
                         <a data-bind="attr: { 'href': $parent.url + hashtagValue }, text: $root.showIds() ? value :  $root.translateQuestion($data)" target="_blank"></a>
+                        <!-- /ko  -->
+                        <!-- ko if: $root.readOnly --><!-- ko text: $root.showIds() ? value :  $root.translateQuestion($data) --><!-- /ko --><!-- /ko -->
```
Comment (emord) [inline corehq/apps/app_manager/templates/app_manager/form_summary_base.html]: Does this mean knockout inserts something like `<p>value</p>` ?

**Review by emord — APPROVED** [review]

File: corehq/apps/app_manager/templates/app_manager/form_summary_base.html (hunk 1 above)
Comment (proteusvacuum) [reply corehq/apps/app_manager/templates/app_manager/form_summary_base.html]: currently, yes: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/views/app_summary.py#L56

File: corehq/apps/app_manager/templates/app_manager/form_summary_base.html (hunk 2 above)
Comment (proteusvacuum) [reply corehq/apps/app_manager/templates/app_manager/form_summary_base.html]: It only inserts `value` as text. I removed all the extra `<span data-bind='text: value'>` from this page as it was drastically affecting performance when there were thousands of questions displayed (and so there were hundreds of thousands of unnecessary nodes on the page).

**Review by orangejenny — APPROVED** [review]

**Pull request merged**